terps/CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ endif()
 # Added glk file prompts for gargoyle using garglk_fileref_get_name.
 #
 if(WITH_ALAN2)
-    set(ALAN2_MACROS GLK)
+    set(ALAN2_MACROS GLK _PROTOTYPES_)
```

Comment:
`_PROTOTYPES_` is already defined in `terps/alan2/sysdep.h`:

```
#ifdef __STDC__
#define _PROTOTYPES_
#include <stdlib.h>
#include <string.h>
#endif
```
This causes a redefinition warning.

`__STDC__` is required to be defined by C compilers; is MSVC, perhaps, not defining it?  If so, I'd think the best thing to do is add something in `sysdep.h` to check for MSVC in addition to `__STDC__`.

I've got a local experimental updated Alan2 interpreter which does away with all this pre-ANSI stuff, so hopefully in the future this won't even be a concern.

Reply:
True, MSVC has a separate compiler option (`/Zc:__STDC__`) that enables this definition. But we don't *want* to have that defined, because doing so would hide away all the POSIX functions.

(MS takes the stance that, since POSIX functions aren't specified by ISO C, they should count as vendor extensions and thus be prefixed with underscores. Which is technically correct, I guess, but very annoying.)